### ISSUE TITLE
Support cipher suites used by Firefox

### ISSUE THREAD

**Issue by krieger-od** [sipwise/rtpengine#141]
There you can find a listing of suites used by recent Firefox:
https://github.com/mozilla/gecko-dev/blob/00bfc9f9d86c2a31b4ce913bbcc39ce1db93b21d/security/manager/ssl/nsNSSComponent.cpp#L648
The concept beyond their chosen list is that only Perfect Forward Secrecy suites are allowed.

And there is a listing of suites supported by Rtpengine:
https://github.com/sipwise/rtpengine/blob/master/daemon/crypto.c#L36

Please add support of any of FF list.

When we make a call with Firefox (39.0 and 42.0a1 was tested), such error appears in rtpengine's log: ```DTLS error: 1 (no shared cipher)```. From browser perspective, this results in no RTP being ever sent, without any warning in its console. The proficient guy from Mozilla reports that it doesn't get reply to its Hello request within DTLS negotiation (see below). So if Rtpengine doesn't send a reply in such case, then this should be also fixed.
```
> Looking at the PCAP, for whatever reason our
> Client Hello is never answered. This could be due to not liking the list of
> cipher suites we offer (although I would expect there to be an error
> response, not silence), or it could be a timing condition of some sort (I've
> looked at the STUN transactions, and they look done by the time we start
> sending Client Hello).
```

Thanks.

**Comment by rfuchs** [comment]
You're comparing SSL/DTLS ciphers with SRTP ciphers. Not the same thing.

"No shared ciphers" should be addressed by 21e1fb680762f421e05fab036f8138b2276f5037

**Comment by krieger-od** [comment]
Sorry for confusion :)
So updating should help.
Thanks a lot!
